src/app/Fake.Core.Process/CreateProcess.fs

```diff
@@ -136,6 +136,23 @@ module CreateProcess =
     let fromRawCommand command args =
         fromCommand <| RawCommand(command, Arguments.OfArgs args)
 
+    /// Create a CreateProcess by using [BlackFox.CommandLine](https://github.com/vbfox/FoxSharp/tree/master/src/BlackFox.CommandLine) library
+    /// 
+    /// ### Example
+    /// 
+    ///     open BlackFox.CommandLine
+    /// 
+    ///     CmdLine.empty
+    ///     |> CmdLine.append "build"
+    ///     |> CmdLine.appendIf noRestore "--no-restore"
+    ///     |> CmdLine.appendPrefixIfSome "--framework" framework
+    ///     |> CmdLine.appendPrefixf "--configuration" "%A" configuration
+    ///     |> CreateProcess.fromCmdLine "dotnet.exe"
+    ///     |> Proc.run
+    ///     |> ignore
+    let fromCmdLine command args =
```

Comment:
@vbfox Maybe that is Okish if we says fake only has build-in APIs for working with argument lists and properly escaping them and other weird stuff (or the builder-API) is external

Reply:
I like that but i'm not 100% objective on the subjects :)

It also suggest that other builder libraries could happen.

If needed I can produce another package with only the escaping part completely separate from the builder one. FAKE core would then depend on it and each FAKE package could use whatever builder it wants.

Not sure it's worth the complexity but it's doable